examples/llm_inference/hugging_face_js/gemma_3n/index.js

```diff
@@ -0,0 +1,387 @@
+// Copyright 2025 The MediaPipe Authors.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//      http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// -------------------------------------------------------------------------- //
+
+import {oauthLoginUrl, oauthHandleRedirectIfPresent} from "@huggingface/hub";
+import {FilesetResolver, LlmInference} from 'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-genai';
```

Comment:
Nit: You can add @mediapipe/tasks-genai to the import map in the index.html if you want consistency between npm style imports and cjs style imports.

Reply:
Done.